freestanding/Makefile

```diff
@@ -1,5 +1,13 @@
-ifneq (, $(shell command -v opam))
-	PKG_CONFIG_PATH ?= $(shell opam var prefix)/lib/pkgconfig
+ifneq (, $(shell command -v opam || which opam))
```

Comment:
Thank you for the fix, but note that the freestanding compilation path will be dropped very soon as it's specific for MirageOS 3.